pyseir/run.py

```diff
@@ -43,15 +43,13 @@ def run(input: OneRegionTimeseriesDataset) -> "OneRegionPipeline":
 
         icu_data = None
 
-        # TODO: Re-enable for CBSAs once typical utilization number aggregation fixed.
-        if input.region.level is not AggregationLevel.CBSA:
-            icu_input = infer_icu.RegionalInput.from_regional_data(input)
-            try:
-                icu_data = infer_icu.get_icu_timeseries_from_regional_input(
-                    icu_input, weight_by=infer_icu.ICUWeightsPath.ONE_MONTH_TRAILING_CASES
-                )
-            except KeyError:
-                _log.exception(f"Failed to run icu data for {input.region}")
+        icu_input = infer_icu.RegionalInput.from_regional_data(input)
+        try:
+            icu_data = infer_icu.get_icu_timeseries_from_regional_input(
+                icu_input, weight_by=infer_icu.ICUWeightsPath.ONE_MONTH_TRAILING_CASES
```

Comment:
Are you sure this works? I don't see CBSAs in data/one_month_trailing_weights_via_fips.json and don't know how they'd get aggregated before this call.

Reply:
yeah that's a good point.  Will leave that out